test/conformance/ingress/update_test.go

```diff
@@ -67,6 +67,7 @@ func TestUpdate(t *testing.T) {
 	defer cancel()
 
 	proberCancel := checkOK(t, "http://"+hostname+".example.com", client)
+	defer proberCancel()
```

Comment:
This is incorrect as it reorders the functions at the end of the function and now the prober continues past the teardown of the final version.  What is the path through this that exits early?